src/coreclr/src/jit/simdcodegenxarch.cpp

```diff
@@ -637,6 +650,46 @@ instruction CodeGen::getOpForSIMDIntrinsic(SIMDIntrinsicID intrinsicId, var_type
             result = INS_insertps;
             break;
 
+        case SIMDIntrinsicCeil:
+            if (compiler->getSIMDSupportLevel() >= SIMD_SSE4_Supported)
+            {
+                if (baseType == TYP_FLOAT)
+                {
```

Comment:
*Nit:* This could be slightly simplified as
```c++
case SIMDIntrinsicCeil:
case SIMDIntrinsicFloor:
    if (compiler->getSIMDSupportLevel() >= SIMD_SSE4_Supported)
    {
        if (baseType == TYP_FLOAT)
        {
            result = INS_roundps;
        }
        else
        {
            assert(baseType == TYP_DOUBLE);
            result = INS_roundpd;
        }

        assert(ival != nullptr);
        *ival = (intrinsicId == SIMDIntrinsicCeil) ? ROUNDPS_TOWARD_POSITIVE_INFINITY_IMM : ROUNDPS_TOWARD_NEGATIVE_INFINITY_IMM;
    }
    break;

Reply:
GitHub supports multiline suggestions now: https://twitter.com/natfriedman/status/1232727363626029056?s=20
See https://github.com/dotnet/runtime/pull/31993/files/a83bb51298fe2b63f6669a43f0b4983ee07015f9#r391776898, for an example

Reply:
@tannergooding Nice! Thanks

Reply:
Applied suggestion. Thanks!